interfaces/builtin/gpio-control.go

```diff
@@ -0,0 +1,49 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016-2017 Canonical Ltd
```

Comment:
2019

Reply:
actually I also renamed file as it was not following convention
`gpio-control.go` -> `gpio_control.go`